collect_app/src/main/java/org/odk/collect/android/application/initialization/ApplicationInitializer.java

```diff
@@ -55,21 +60,30 @@ public void initialize() {
         initializeLocale();
     }
 
-    public void initializePreferences() {
+    private void initializePreferences() {
         performMigrations();
         reloadSharedPreferences();
     }
 
-    public void initializeFrameworks() {
+    private void initializeFrameworks() {
         NotificationUtils.createNotificationChannel(context);
         JodaTimeAndroid.init(context);
         initializeLogging();
         AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
         initializeMapFrameworks();
         initializeJavaRosa();
+        initializeAnalytics();
     }
 
-    public void initializeLocale() {
+    private void initializeAnalytics() {
+        String formUpdateMode = generalSharedPreferences
+                .getSharedPreferences()
+                .getString(GeneralKeys.KEY_FORM_UPDATE_MODE, FormUpdateMode.MANUAL.getValue(context));
+
+        analytics.setUserProperty("FormUpdateMode", formUpdateMode);
```

Comment:
I wanted to do this here rather than when the settings are changed/imported as it avoids handling all the different possible cases and also makes sure we always have a value for this property set. It means the values we have are probably going to be slightly behind (as we'll be waiting for the next time the app boots for the property to be set) but I think that'll be ok.